.github/labeler.yml

```
@@ -0,0 +1,38 @@
+# Configuration for GitHub Action `.github/workflows/pull-request-labeler.yml`.
+# See https://github.com/marketplace/actions/labeler for more information.
+
+# Only documentation changes
+docs:
+  - all: [ '**/*.md' ]
+
+# Only test changes
+dev/testing:
+  - all: [ '{spec/**,integration/**}' ]
```

Minor: Small additions 😄 

```suggestion
dev/testing:
  - all: [ '{spec/**,integration/**,benchmarks/**,sorbet/**}' ]
```